src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
It seems like there is a syntax error due to a missing closing quotation mark at the end of the string. Make sure to close the string with the appropriate quotation mark to fix the syntax error.